packages/js/data/src/index.ts

```diff
@@ -16,6 +16,7 @@ export { NAVIGATION_STORE_NAME } from './navigation';
 export { OPTIONS_STORE_NAME } from './options';
 export { ITEMS_STORE_NAME } from './items';
 export { PAYMENT_GATEWAYS_STORE_NAME } from './payment-gateways';
+export { PAYMENT_GATEWAYS_SUGGESTIONS_STORE_NAME } from './payment-gateway-suggestions';
```

Comment:
I think we'll probably also want to use the same data store to manage payment settings that are changed on the page in future, for example when payment gateways are enabled/disabled/reordered we can fire those events through WP data. What do you think about the store name being `payment_settings`, so it's more generic? Pulling payment gateway suggestions is one part of it 🙂 

I'm not sure on best practice here, so pinging @woocommerce/ghidorah for thoughts.

Reply:
I agree with @dmallory42 that we can use a more generic data store name like payment_settings. However, I wonder if we could extend the existing payment gateway data store instead of creating a new one. What do you think?

Reply:
changed in https://github.com/woocommerce/woocommerce/pull/52824/commits/69f51b99ac41b32f4f67bbea89de6ddd68d2791e https://github.com/woocommerce/woocommerce/pull/52824/commits/0c3a8ff24f7b5aefa273b596ea5335b1939cf386

Reply:
I agree with your conclusion about not mixing them because of the different data structures, @oaratovskyi. These changes LGTM!